packages/ui/components/calendar/components/Calendar.tsx

```diff
@@ -0,0 +1,135 @@
+            containerRef={container}
+          /> */}
+            <div className="flex flex-auto">
+              <div className="sticky left-0 z-10 w-14 flex-none bg-white ring-1 ring-gray-100" />
+              <div className="grid flex-auto grid-cols-1 grid-rows-1 ">
+                <HorizontalLines
+                  hours={hours}
+                  numberOfGridStopsPerCell={usersCellsStopsPerHour}
+                  containerOffsetRef={containerOffset}
+                />
+                <VeritcalLines days={days} />
+
+                {/* Empty Cells */}
+                <SchedulerColumns
+                  zIndex={50}
+                  offsetHeight={containerOffset.current?.offsetHeight}
+                  gridStopsPerDay={numberOfGridStopsPerDay}>
+                  <>
+                    {[...Array(days.length)].map((_, i) => (
+                      <li
+                        key={i}
+                        style={{
+                          gridRow: `2 / span ${numberOfGridStopsPerDay}`,
+                          position: "relative",
+                        }}>
+                        {/* While startDate < endDate:  */}
+                        {[...Array(numberOfGridStopsPerDay)].map((_, j) => {
+                          const key = `${i}-${j}`;
+                          return (
+                            <EmptyCell
+                              key={key}
+                              day={days[i].toDate()}
+                              gridCellIdx={j}
+                              totalGridCells={numberOfGridStopsPerDay}
+                              selectionLength={endHour - startHour}
+                              startHour={startHour}
+                            />
+                          );
+                        })}
+                      </li>
+                    ))}
+                  </>
+                </SchedulerColumns>
+
+                <SchedulerColumns
+                  offsetHeight={containerOffset.current?.offsetHeight}
+                  gridStopsPerDay={numberOfGridStopsPerDay}>
+                  {/*Loop over events per day  */}
+                  {days.map((day, i) => {
+                    return (
+                      <li key={day.toISOString()} className="relative" style={{ gridColumnStart: i + 1 }}>
+                        <EventList day={day} />
+                        <BlockedList day={day} containerRef={container} />
+                      </li>
+                    );
+                  })}
+                </SchedulerColumns>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </MobileNotSupported>
+  );
+}
+
+/** @todo Will be removed once we have mobile support */
+const MobileNotSupported = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <>
+      <div className="flex h-full flex-col items-center justify-center sm:hidden">
+        <h1 className="text-2xl font-bold">Mobile not supported</h1>
```

Comment:
Stupid NIT, but perhaps we should add "yet" -> Mobile not supported yet. Might prevent people from getting disappointed by assuming we're not going to build this 😄

Reply:
the plan is to not actually render this on mobile at all but we can for sure add this